qiskit_nature/properties/second_quantization/electronic/bases/__init__.py

```diff
@@ -25,6 +25,16 @@
 
 from .electronic_basis import ElectronicBasis
 from .electronic_basis_transform import ElectronicBasisTransform
+from .....deprecation import warn_deprecated, DeprecatedType, NatureDeprecationWarning
+
+warn_deprecated(
+    "0.5.0",
+    old_type=DeprecatedType.PACKAGE,
+    old_name="qiskit_nature.properties.second_quantization.electronic.basis",
+    new_type=DeprecatedType.PACKAGE,
+    new_name="qiskit_nature.second_q.properties.basis",
```

Comment:
After the latest design plans (yet to be published here on github), this will actually go here instead:
```suggestion
    new_name="qiskit_nature.second_q.problems.bases",
```